packages/web-react/src/components/Button/demo/createButtonDemo.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Button } from '../Button';
+import { Icon } from '../../Icon';
+import { ButtonBaseProps } from '../../../types';
+
+const createButtonDemo = (labels: [string, string], props?: ButtonBaseProps) => {
```

Comment:
What about to make a hook from this? `useDemoButtons`? Not quite sure about this but it seems fine to me :-) What do you think?

Reply:
I always thought that returning JSX from hooks is an antipattern. But I can rework it.

Reply:
No, you are right. In fact, this is a "factory" function, see https://github.com/themithy/react-design-patterns/blob/master/doc/factory-pattern.md. So I am thinking now just about the naming if we should use `factory` in the name or `render` or something else. 🤔

Reply:
Created new component `<ButtonDemoFactory />`, its better than the function before, thank you.